spec/jobs/partnership_activation_job_spec.rb

```diff
@@ -2,36 +2,44 @@
 
 require "rails_helper"
 
-RSpec.describe "PartnershipFinalisationJob" do
+RSpec.describe PartnershipActivationJob do
   describe "#perform" do
-    let(:pending_partnership) { create(:partnership, :pending) }
+    let(:partnership) { create(:partnership, :pending) }
+    let(:report_id) { partnership.report_id }
+
     let!(:school_cohort) do
       SchoolCohort.create!(
-        school: pending_partnership.school,
-        cohort: pending_partnership.cohort,
+        school: partnership.school,
+        cohort: partnership.cohort,
         induction_programme_choice: "core_induction_programme",
       )
     end
 
+    def execute
+      subject.perform(partnership: partnership, report_id: report_id)
+    end
+
     it "updates the partnership and school cohort" do
-      PartnershipActivationJob.new.perform(pending_partnership)
+      execute
 
-      expect(pending_partnership.reload.pending).to eql false
+      expect(partnership.reload.pending).to eql false
       expect(school_cohort.reload.induction_programme_choice).to eql "full_induction_programme"
     end
 
-    it "does nothing when the partnership request has been challenged" do
-      challenged_partnership = create(:partnership, :pending, :challenged)
-      school_cohort = SchoolCohort.create!(
-        school: challenged_partnership.school,
-        cohort: challenged_partnership.cohort,
-        induction_programme_choice: "core_induction_programme",
-      )
+    context "when aprtnership has been challenged" do
```

Comment:
Typo 😛

Reply:
"You don't know what aprtnership means??" :P